crates/ruma-common/src/events/message.rs

```diff
@@ -1,7 +1,52 @@
 //! Types for extensible text message events ([MSC1767]).
 //!
+//! # Extensible events
+//!
+//! MSCs [1767] (Text, Emote and Notice), [3551] (Files), [3552] (Images and Stickers), [3553]
+//! (Videos), [3246] (Audio), and [3488] (Location) introduce new primary types called extensible
+//! events. These types are meant to replace the `m.room.message` primary type and its `msgtype`s.
+//! Other MSCs introduce new types with an `m.room.message` fallback, like [MSC3245] (Voice
+//! Messages), and types that only have an extensible events format, like [MSC3381] (Polls).
+//!
+//! # Transition Period
+//!
+//! MSC1767 defines a transition period that will start after the extensible events are released in
+//! a Matrix version. It should last approximately one year, but the end of that period will be
+//! officialised in a new Matrix version.
```

Comment:
"officialised" is a weird word and looks like British English (the rest of Ruma is AE for consistency reasons). Would be nice to come up with sth. different.